pcsx2/Patch_Memory.cpp

```diff
@@ -249,41 +249,253 @@ void handle_extended_t(IniPatch *p)
 		}
 		else if (p->addr < 0xE0000000)
 		{
-			if (((u32)p->data & 0xFFFF0000) == 0x00000000)		// Daaaaaaa 0000dddd
+			if (((u32)p->data & 0x00F00000) == 0x00000000)		// Daaaaaaa yy0zdddd
 			{
-				u16 mem = memRead16((u32)p->addr & 0x0FFFFFFF);
-				if (mem != (0x0000FFFF & (u32)p->data))
+				u8 z = ((u32)p->data & 0x000F0000) / 0x10000;
```

Comment:
Can be constant, same for the others.

Reply:
Also make it a shift rather than a divide, there's no need for it to be a divide and it's less clear.

Reply:
Yeah, I considered doing that as well, but since the whole rest of the file used dividing instead of shifting, I also went with it to keep it consistent.

I'll get around to that hopefully by the weekend when I have some more tree time.

Reply:
You're welcome to improve the others :D